### PR TITLE
NL ASR model (seq2seq) fine tuned wav2vec2 - CommonVoice7.0

### PR DESCRIPTION
**Pull request by sachinsachdeva**
Hello fellow SpechBrainers :) ,

First of all thanks building and making is awesome ToolKit. I have been playing around with the ASR part (and related recipes ) for  year and finally have something useful to contribute :).

This PR has the finetuned Dutch recipe and the corresponding model.
The recipe is similar to the English counterpart. 
Following 2 things are still pending ( I can use some help )
1. Where exactly to upload the trained model (The others I have looked are in @mravanelli )
2. How to make the trained model compatible with HugginFace ?

Once I can figure the above 2 things I can finalize the PR.

### CODE REVIEW

**Comment by TParcollet** [comment]
Hi and thanks !

I see that you decided to go for BPE and seq2seq. For some reasons, w2v2-based ASR models tend to work better with pure CTC and characters (it's also much faster to train), did you try it as well ?

**Comment by sachinsachdeva** [comment]
Just ran the CTC only with char and definitely better results, do you want me to create a separate PR for that or add to this one ?
Also I did try unigram with seq2seq, but the results were considerably worst (adding result below from that run).
Test CER of 29.39 and test WER of 39.42

**Comment by TParcollet** [comment]
No you can add it directly here :-)

**Comment by TParcollet** [comment]
Hi, to upload it to huggingface, you should open a PR on the huggingface hub. To transform it to this, just take example from the other model that we have on the hub and that are wav2vec2 as well (the french one for CV FR for instance). You will have to change a bit the YAML and rename some ckpt files to match the names that you give in the yaml. If you are struggling, I can help :-) Would be great if you can give us a link pointing to the log folder (from anywhere) and we will download it and upload it :-)

File: templates/audio_mnist/train.yaml [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+# If you plan to train a system on an HPC cluster with a big dataset,
+# we strongly suggest doing the following:
+# 1- Compress the dataset in a single tar or zip file.
+# 2- Copy your dataset locally (i.e., the local disk of the computing node).
+# 3- Uncompress the dataset in the local folder.
+# 4- Set data_folder with the local path.
+# Reading data from the local disk of the compute node (e.g. $SLURM_TMPDIR with SLURM-based clusters) is very important.
+# It allows you to read the data much faster without slowing down the shared filesystem.
+data_folder: /Users/sachinsachdeva/ws/kaggle/datasets
```
Comment (anautsch) [inline templates/audio_mnist/train.yaml:21]: As you did in the other yaml files, please use a `!PLACEHOLDER` here

**Review by anautsch — COMMENTED** [review]
Hi @sachinsachdeva thank you for the PR!

Was there progress on the model uploading to HugginFace or on this PR?
(I saw the 3-months silence for this discussion; sorry for that.)

**Comment by anautsch** [comment]
@sachinsachdeva please split this PR into two PRs
* CommonVoice recipe (here; title suggests it already)
* new template

The Dutch recipe should be ok. The template might have some discussions to it (mself I do not know where to draw the boundary between a "recipe" and a "template" — we just need operational definitions there, for that we need to discuss – which shouldn't hamper the release of your wav2vec2 contribution).

**Comment by sachinsachdeva** [comment]
Hello @anautsch , thanks a lot for the feedback. unfortunately currently I don't have access to my Deep Learning PC (where all the trained models are) and after looking at your feedback to split this into 2 PRs , I think best is to close this PR as it is and when I have access to my PC again I can create the appropriate PRs along with model uploads

**Comment by anautsch** [comment]
@sachinsachdeva sorry to hear this. As you are the contributor, I can but moderate.
There has been work done here; it does not seem right to let it all slip - when you get back to this, please do let us know.

As you asked, I am closing this PR.
Please re-open again, when you get back.

**Pull request closed without merging**